File: array/rotate.py
#!/usr/bin/python

"""
Rotate given array by k position
"""
def rotate(arr, k):
  n = len(arr)
  for i in range(k):
    tmp = arr[0]
    for j in range(n - 1):
      arr[j] = arr[j + 1]
    arr[n - 1] = tmp

File: array/rotate.js
function rotate(arr, k) {
  let n = arr.length;
  for (let i = 0; i < k; i++) {
    let tmp = arr[0];
    for (let j = 0; j < n - 1; j++) {
      arr[j] = arr[j + 1];
    }
    arr[n - 1] = tmp;
  }
}
